cpp/src/slice2swift/Gen.cpp

```diff
@@ -1082,7 +1073,9 @@ Gen::ProxyVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
     out << nl << "///    - communicator: The communicator of the new proxy.";
     out << nl << "///    - proxyString: The proxy string to parse.";
     out << nl << "///    - type: The type of the new proxy.";
+    out << nl << "///";
```

Comment:
There should be newlines between different sections (Parameters, Throws, and Returns).